spec/validators/expiry_date_validator_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe ExpiryDateValidator, type: :validator do
+  let(:validator) { described_class.new }
```

Comment:
Small nitpick : with the refactoring, this is now unused as the Bet model is using your validator under the hood when testing for valid or not, otherwise :+1: